enigma/src/main/java/org/quiltmc/enigma/api/translation/representation/entry/MethodEntry.java

```diff
@@ -49,29 +50,45 @@ public boolean isConstructor() {
 	}
 
 	/**
-	 * Creates an iterator of all parameters in this method. Unmapped args will have no name, and javadoc is ignored.
+	 * Get a list of all parameters in this method.
```

Comment:
```suggestion
	 * Get a list of all parameters in this method. All parameters will have an empty name.
```
I think the clarification is important (read: I will forget and wonder why my code doesn't work)